service/service.go

```diff
@@ -110,6 +113,29 @@ func (svc *Service) Close(ctx context.Context) error {
 	return nil
 }
 
-func registerCheckers(ctx context.Context, hc HealthChecker) error {
+func (svc *Service) registerCheckers() error {
+	hc := svc.HealthCheck
+
```

Comment:
dunno what sort of error logging we should have here

Reply:
The return-all-the-way-up-the-call-stack-and-log-once kind!